spaceR/spaceR/ViewModels/LoginViewModel.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  LoginViewModel.swift
+//  SpaceR
+//
+//  Created by Kieran Woodrow on 2022/03/13.
+//
+
+import Foundation
+import UIKit
+
+class LoginViewModel {
+    
+    private var user: [User] = []
+    // swiftlint:disable force_cast
+    private let coreDataPersistantObject = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
+    // swiftlint:enable force_cast
+    
+    func validateUserCredentials(userEmail: String, userPassword: String) -> Bool {
+        if validateUserEmailInLoginProcess(userEmail: userEmail) &&
+            validateUserPasswordInLoginProcess(userPassword: userPassword) {
+            return true
+        } else {
+            return false
+        }
+    }
+    
+    func validateUserEmailInLoginProcess(userEmail: String) -> Bool {
```

Comment:
So this function name is a little bit long try making the names of your functions a little but shorter maybe something like: 

`valid(userEmail: String) -> Bool`

Reply:
How aboput validateUserEmail? (Again, the idea of the function name being explicit in what it is doing)

Reply:
See above comment 😄